### ISSUE TITLE
NameError: name 'compute_privacy' is not defined

### ISSUE THREAD

**Issue by Sherrylife** [Antonious-Grigis/Shuffled-Model-of-Differential-Privacy-in-Federated-Learning#1]
Hi，@Antonious-Grigis， when I run the code 'MNIST_dpsgd_norm_infinity.py', I receive the error in picture, and your code seems incomplete. Can you take the time to fix it?
![image](https://user-images.githubusercontent.com/60345931/190854622-0a70c8fb-2ebb-47d3-b26b-3b736e26dd64.png)

**Comment by Sherrylife** [comment]
Another question is about the annotation of the variable 'eps0' in your code? According to the definition 1 in your original paper, 'eps0' is Local Differential Privacy not the Central Differential Privacy, but in your code you claim that 'eps0' is Central Differential Privacy,  can you explain it? 
![image](https://user-images.githubusercontent.com/60345931/190856785-208bb63d-eb06-43d7-874b-4fc280858129.png)
![image](https://user-images.githubusercontent.com/60345931/190856797-86d50afa-3159-4650-9fc0-d6e80c6e4a11.png)

**Comment by Antonious-Grigis** [comment]
Thank you for your question!
'compute_privacy' is a function to compute the central privacy parameter eps for given the epoch number and system parameters.
Recently, we have a new result to compute a tighter central DP parameter eps for the sub-sampled shuffled model in NeurIPS 2021. I have uploaded a file for privacy computation using this new result. 
You are right, it is a typo. It should be 'local privacy level'. I have updated it. 

Thanks!
